templates/web/base/admin/reports/edit.html

```diff
@@ -131,7 +131,22 @@
 <li><label class="inline-text" for="category">[% loc('Category:') %]</label>
     [% INCLUDE 'admin/report-category.html' %]
 </li>
-<li>[% loc('Extra data:') %] [% IF extra_fields.size %]<ul>[% FOREACH field IN extra_fields %]<li><strong>[% field.name %]</strong>: [% field.val %]</li>[% END %]</ul>[% ELSE %]No[% END %]</li>
+<li>[% loc('Extra data:') ~%]
+    [%~ IF extra_fields.size ~%]
+        <ul>
+        [%~ FOREACH field IN extra_fields ~%]
+            <li><strong>[%~ field.name ~%]</strong>: [% IF field.name == loc("Created By") ~%]
+                    [%# We can't use the .list.join() approach here because field.val
```

Comment:
Instead of testing for a specific field, how about `IF field.val.0.defined` which returns true if field.val is a list (unless its first item is undef, I guess!) and then you can do the join, and just print it out if false.

Reply:
I knew there must be a more elegant way of doing it! Thanks, that's much neater.